### DISCUSSION TITLE
integration

### DISCUSSION THREAD

**Discussion by rakibunateed** [faisal-gits/bKash-payment-gateway-API-integration#6] [Q&A] [answered]
bKash payment gateway API integration by using php

**Comment by faisal-gits** [comment, marked as answer]
Please refer to our extensive documentation for more information.

**Comment by rakibunateed** [comment]
is this official?

**Comment by faisal-gits** [comment]
unofficial guide about integrating bKash payment gateway in your application

**Comment by faisal-gits** [comment]
Official Links https://developer.bka.sh/reference/createpaymentusingpost
